backend/database/models/officer.py

```diff
@@ -86,14 +86,34 @@ def __repr__(self):
         return f"<StateID {self.id}>"
 
 
+class Employment(db.Model):
```

Comment:
You have this defined twice; here and in `employment.py`

Reply:
Nice catch. Thanks!